### ISSUE TITLE
test(training): prove pyFDN temporal sketch conditioning end to end

### ISSUE THREAD

**Issue by ktinubu** [tinaudio/synth-setter#3119]
## Goal

Prove the production pyFDN temporal-sketch path without mocks, fakes, monkeypatching, synthetic checkpoints, or hand-written production artifacts.

## Required path

```text
real pyFDN dataset generation
→ production Lance artifact
→ production pyfdn_sketch augmentation
→ real datamodule
→ real sketch tokenizer and training step
→ checkpoint save/reload
→ conditional parameter sampling
```

## Acceptance criteria

- [ ] Original rows, audio, and parameters survive augmentation.
- [ ] Stored sketch schema is exactly EDC `[8,32]`, echo density `[32]`, and spectral flatness `[32]`.
- [ ] The model consumes finite `float32[B,10,32]` and emits 32 temporal control tokens.
- [ ] Training loss and gradients are finite.
- [ ] A real training-produced checkpoint reloads and samples finite pyFDN parameter predictions.

Blocked by the extraction, temporal-tokenization, and Lance-storage tasks.
Part of #3002.
